src/sentry/static/sentry/app/components/asyncComponent.tsx

```diff
@@ -13,7 +13,25 @@ import PermissionDenied from 'app/views/permissionDenied';
 import RouteError from 'app/views/routeError';
 import getRouteStringFromRoutes from 'app/utils/getRouteStringFromRoutes';
 
-export default class AsyncComponent extends React.Component {
+type Props = {
+  location?: any;
```

Comment:
Should this use the [Location interface](https://github.com/Microsoft/TypeScript/blob/master/lib/lib.dom.d.ts#L9516)?

Reply:
Be aware that the `location` object may be sourced from instances of react-router v3.